keras_cv/models/object_detection/retina_net/__internal__/layers/prediction_head.py

```diff
@@ -30,7 +30,7 @@ class PredictionHead(layers.Layer):
         or the box regression head depending on `output_filters`.
     """
 
-    def __init__(self, output_filters, bias_initializer, **kwargs):
+    def __init__(self, output_filters, bias_initializer, conv_layers=1, **kwargs):
```

Comment:
Note that the default value is changed from 4 to 1, so this might affect some existing prod code (if any)

Reply:
there is no prod code, but there are actually greater issues here 😨 the integration tests fail if you set `prediction_head.trainable=False`, but if trainable they fail.